.nxignore

```diff
@@ -3,3 +3,4 @@ packages/@ama-sdk/generator-sdk/src/**/templates/**/*.{j,t}s
 packages/@*/*/{builders,schematics}/**/templates/**/*.ts
 packages/@*/*/{builders,schematics}/**/mocks/**/*.ts
 packages/@*/*/build/**
+!**/schematics/**/target
```

Comment:
Could not be done in a file at `packages/@ama-sdk/schematics/.nxignore` ?

Reply:
I didn't try but according to the docs https://nx.dev/reference/nxignore, this file is only expected at root level